main.css

```
@@ -0,0 +1,164 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 1px solid #DFDFDF;
+  background-color: #FFFFFF;
+  height: 70px ;
+  padding: 0 200px;
+}
+
+.loginBtn {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #3692FF;
+  color: #FFFFFF;
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 19.09px;
+  padding: 12px 20px;
+  border-radius: 8px;
+  gap: 10px;
+  text-decoration: none;
+}
+
+.loginBtn:hover {
+  background-color: #1967D6;
+}
+
+.wrapper {
+  max-width: 1200px;
+  margin: 0 auto;
+  width: 100%;
+}
+
+.banner {
+  height: 540px;
+  background-color: #CFE5FF;
+  display: flex;
+  align-items: center;
+  background-image: url('./img/home/hero-image.png');
+  background-repeat: no-repeat;  
+  background-size: 50%;
+  background-position: 74% bottom;
+}
+
+.trade {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 56px;
+  color: #374151;
+}
+
+.watch {
+  font-size: 20px;
+  font-weight: 600;
+  line-height: 24px;
+  text-align: center;
+  color: #FFFFFF;
+  padding: 16px 124px;
+  border-radius: 40px;
+  background-color: #3692FF;
+  display: inline-flex;
+  justify-content: center;
+  text-decoration: none;
+  margin-top: 32px;
+}
+
+.watch:hover {
+  background-color: #1967D6;
+}
+
+.product {
+  display: flex;
+  align-items: center;
+  padding: 138px 0;
+  gap: 64px;
+}
+
+.product-image {
+  flex: 1;
+}
+
+.product-details {
+  flex: 1;
+}
+
+.product:nth-child(2) {
+  text-align: right;
+}
+
+
+.product-content-tag {
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 25.2px;
+  color: #3692FF;
+}
+
+.product-content-title {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 56px;
+  color: #374151;
+}
+
+.product-content-description {
+  font-size: 24px;
+  font-weight: 500;
+  line-height: 28.8px;
+  color: #374151;
+  letter-spacing: 8%;
+  margin-top: 24px;
+}
+
+#bottom-banner {
+  background-image: url('./img/home/bottom-banner-image.png');
+}
+
+.bottom-banner-title {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 56px;
+  color: #374151;
+}
+
+footer {
+  padding: 32px 200px 108px 200px;
```

아래와 같이 축약형으로 작성할 수 있어요!
```suggestion
  padding: 32px 200px 108px;
```